packages/devtools-reps/src/launchpad/components/Console.css

```diff
@@ -38,6 +38,11 @@ main {
   margin-bottom: 0.5rem;
 }
 
+.rep-element {
```

Comment:
This aligns the styling to the webconsole's. The newlines were being ignored before.